template.md

```diff
@@ -137,6 +133,52 @@ Bold entries here are especially relevant to blockchain projects.
 #### [zkSync](https://github.com/matter-labs/zksync)
 
 
+&nbsp;
+
+### Rust in Bitcoin
+
+#### [Rust Bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+
+#### [BDK](https://github.com/bitcoindevkit/bdk)
+
+#### [LDK](https://github.com/lightningdevkit/rust-lightning)
+
+#### [Sapio](https://github.com/sapio-lang/sapio)
+
+#### [LNP/BP](https://github.com/LNP-BP)
+
+#### [RGB](https://github.com/rgb-org)
+
+#### [Internet2](https://github.com/internet2-org/)
```

Comment:
This link doesn't work. This org appears to be called internet2-WG now. Can you fix it?

Reply:
Updated